src/auth/firebase.ts

```diff
@@ -0,0 +1,24 @@
+import { initializeApp } from 'firebase/app';
+import {
+  getAuth,
+  GoogleAuthProvider,
+  OAuthProvider,
+  GithubAuthProvider,
+} from 'firebase/auth';
+
+const firebaseConfig = {
```

Comment:
It would be better to put it in env but that would require extra fiddling with deployment